python/paddle/fluid/tests/unittests/prim_op_test.py

```diff
@@ -675,6 +680,19 @@ def check_jit_comp(self):
         )
         net = PrimNet(self.public_python_api)
         net = apply_to_static(net, False)
+        # ensure the operator not in program if check_prim is True
+        # because to_static will build backward graph in forward, fill_any_like op's
+        # backward op is itself, so we skip it here.
+        if self.op_type != "fill_any_like":
+            forward_ops = [
+                op.type
+                for op in net.forward.get_concrete_program(args)[1]
+                ._train_program.block(0)
+                .ops
+            ]
```

Comment:
use net.forward.get_concrete_program(args)[1].forward_program to avoid 'fill_any_like' condition

Reply:
done